### DISCUSSION TITLE
articles/linux/install_and_config

### DISCUSSION THREAD

**Discussion by giscus** [lxl66566/lxl66566.github.io#18] [General]
# articles/linux/install_and_config

安装与配置

https://absx.pages.dev/articles/linux/install_and_config.html

**Comment by ygnid** [comment]
v2raya的lite模式和非lite模式有啥区别吗, 我看更新的时候说非lite模式要被弃用了, 另外为啥要切换到daed?

**Reply by lxl66566** [reply]
lite 不需要 root 权限，不能开透明代理；换 daed 只是听 archlinuxcn 群友提到，然后尝尝新（

**Reply by ygnid** [reply]
好吧, 我还是继续用v2raya, 能不折腾就不折腾

**Reply by Ic-arbon** [reply]
都用arch了，包折腾的
